components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -4444,13 +4655,171 @@ components:
         orgHandle:
           type: string
           example: 'abcbuilders'
+        parentId:
+          type: string
+          example: '08f8c1d2-4b3e-4c5a-9f6b-7d8e9f0a1b2c'
+          description: |
+              The parent organization ID.
         status:
           type: string
           enum: [ ACTIVE, DISABLED ]
           example: ACTIVE
         ref:
           type: string
           example: '/t/wso2.com/api/server/v1/organizations/b4526d91-a8bf-43d2-8b14-c548cf73065b'
+        hasChildren:
+          type: boolean
+          example: true
+        depthFromRoot:
```

Comment:
Please make sure that the value properly return based on the root starting level of the org structure